indexer/database/blocks.go

```diff
@@ -21,6 +23,15 @@ type BlockHeader struct {
 	Timestamp  uint64
 }
 
+func BlockHeaderFromHeader(header *types.Header) BlockHeader {
```

Comment:
```suggestion
func BlockHeaderFromGethHeader(header *types.Header) BlockHeader {
```
More explicit what this means?

Reply:
sgtm